deploy/gk-deploy

```diff
@@ -479,7 +479,20 @@ else
   output "heketi is now running."
 fi
 
-eval_output "heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY} 2>&1"
-if [[ $? -ne 0 ]]; then
-  output "Please check the failed node or device and restart with --load"
+load_temp=$(mktemp)
+eval_output "heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY} 2>&1" | tee ${load_temp}
+grep -q "Unable" ${load_temp}
+unable=$?
+rm ${load_temp}
+
+if [[ ${PIPESTATUS[0]} -ne 0 ]] || [[ ${unable} -eq 0 ]]; then
```

Comment:
It's one of those sneaky internal BASH variables. :) From the [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/internalvariables.html):

> $PIPESTATUS
>     Array variable holding exit status(es) of last executed foreground pipe.
> 
> ```
>     bash$ echo $PIPESTATUS
>     0
> 
>     bash$ ls -al | bogus_command
>     bash: bogus_command: command not found
>     bash$ echo ${PIPESTATUS[1]}
>     127
> 
>     bash$ ls -al | bogus_command
>     bash: bogus_command: command not found
>     bash$ echo $?
>     127
> ```
>The members of the $PIPESTATUS array hold the exit status of each respective command executed in a pipe. $PIPESTATUS[0] holds the exit status of the first command in the pipe, $PIPESTATUS[1] the exit status of the second command, and so on.